crates/oxc_minifier/src/ast_passes/peephole_replace_known_methods.rs

```diff
@@ -446,8 +479,10 @@ mod test {
         fold_same("x = 'abcde'.charCodeAt(5)"); // or x = (0/0)
         fold_same("x = 'abcde'.charCodeAt(-1)"); // or x = (0/0)
         fold_same("x = 'abcde'.charCodeAt(y)");
-        fold_same("x = 'abcde'.charCodeAt()"); // or x = 97
-        fold_same("x = 'abcde'.charCodeAt(0, ++z)"); // or (++z, 97)
+        // START: note the following test cases outputs differ from Google's
+        fold("x = 'abcde'.charCodeAt()", "x = 97");
+        fold("x = 'abcde'.charCodeAt(0, ++z)", "x = 97");
```

Comment:
This does not appear correct. `z++` is missing.

Reply:
fixed